src/main/kotlin/io/thecontext/ci/output/MarkdownRenderer.kt

```diff
@@ -0,0 +1,17 @@
+package io.thecontext.ci.output
+
+import org.commonmark.parser.Parser
+import org.commonmark.renderer.html.HtmlRenderer
+
+interface MarkdownRenderer {
```

Comment:
Name is confusing, because it actually renders HTML, isn't it? So it should be called `HtmlRenderer`?

Reply:
nit: Will there ever be another implementation but the default one?